src/pkg/packager/unload.go

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2021-Present The Zarf Authors
+
+// Package packager contains functions for interacting with, managing and deploying Zarf packages.
+package packager
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/defenseunicorns/zarf/src/config"
+	"github.com/defenseunicorns/zarf/src/pkg/message"
+	"github.com/defenseunicorns/zarf/src/pkg/utils"
+	"github.com/defenseunicorns/zarf/src/pkg/utils/helpers"
+	"github.com/defenseunicorns/zarf/src/types"
+)
+
+// Unload pulls resources from a package (images, git repositories, etc) and pushes them to remotes in the air gap without deploying them
+func (p *Packager) Unload(packageName string) (err error) {
+	spinner := message.NewProgressSpinner("Unloading zarf package %s", packageName)
```

Comment:
Unsure how this is intended to be called, but we should probably use the package name from the `p *Packager` metadata.

Reply:
yep yep - refactored